core-web/libs/edit-content/src/lib/fields/dot-edit-content-binary-field/dot-edit-content-binary-field.component.ts

```diff
@@ -99,15 +112,33 @@ export class DotEditContentBinaryFieldComponent
     readonly BinaryFieldStatus = BinaryFieldStatus;
     readonly BinaryFieldMode = BinaryFieldMode;
     readonly vm$ = this.dotBinaryFieldStore.vm$;
-    private tempId = '';
     dialogOpen = false;
+    customMonacoOptions: Signal<MonacoEditorConstructionOptions> = computed(() => {
+        return {
+            ...this.parseCustomMonacoOptions(this.contentTypeField().fieldVariables)
+        };
+    });
+    private onChange: (value: string) => void;
+    private onTouched: () => void;
+    private tempId = '';
 
-    private get variable(): string {
-        return this.field.variable;
+    constructor(
+        private readonly dotBinaryFieldStore: DotBinaryFieldStore,
+        private readonly dotMessageService: DotMessageService,
+        private readonly dotBinaryFieldEditImageService: DotBinaryFieldEditImageService,
+        private readonly DotBinaryFieldValidatorService: DotBinaryFieldValidatorService,
+        private readonly cd: ChangeDetectorRef
+    ) {
+        this.dotMessageService.init();
+    }
+
+    @Input({ required: true })
+    set field(contentTypeField: DotCMSContentTypeField) {
```

Comment:
Shouldn't `@Input()` be before the `constructor`?

Reply:
is by the 'set' the link order that in that order no idea.